cli/src/commands/upload.ts

```diff
@@ -39,7 +40,9 @@ export const uploadCommand = new Command('upload')
 
     const provider = await makeProviderFromNodeURLAndSecret(globalOptions)
 
-    const chunkSize = parseInt(options.chunkSize)
+    // set chunksize from options or config
+    const chunkSize =
+      parseInt(options.chunkSize) || (globalOptions.chunk_size as number)
```

Comment:
Just noticed an issue with this code.

As `options.chunkSize` is defied as follow:
```
.option(
    '-c, --chunk-size <size>',
    'Chunk size in bytes',
    DEFAULT_CHUNK_SIZE.toString()
  )
```

There will ALWAYS be a value. 

Also, your `||` will not check if `option.chunkSize` is defined or not, but if the output of `parseInt` is defined or not. And `parseInt` always returns a number, so that's not the right way to do this.

IMO, we should remove the default value of the option, and modify the code like so:
```suggestion
    try {
        const chunkSize =
          BigInt(options.chunkSize || globalOptions.chunk_size)
    } catch (e) {
        throw new Error(`failed to parse chunk_size: ${e}`)
    }
```


⚠️ Please note that this issue might also occur on other options such as the globals ones

Reply:
Regarding your comment, only the node_url should be a problem in terms of user experience. I suggest that since this pram is never empty, we prioritize what is entered in the config. In the docs we should add a comment as such: ```node_url:"" // Keep empty if you want to use default value

Reply:
btw ```chunkSize```is a number

Reply:
thinking twice the solution you suggest forces the user to 1) set the chunkSize at each upload or have a commandFile... the advantage of having a default value was the ease of use of the command with straight forward args... could we imagine another system ?